### ISSUE TITLE
[Bug]: Music does not play in multiplayer if Genesis is installed

### ISSUE THREAD

**Issue by nocturni** [Zepalesque/Aether-Redux-Issue-Tracker#128]
### Minecraft Version

1.20.1

### Mod Version

2.0.16c

### Forge/Neoforge Version

Neoforge 47.1.105

### What happened?

Music does not play at all

### What was the expected result?

Music plays properly

### Steps to Reproduce

1. Get Genesis and enable nighttime tracks
2. Connect to a server that has The Aether, Genesis and Redux installed
3. Music does not play unless nighttime tracks are disabled

### Is this a compatibility issue with another mod?

The Aether: Genesis

### Additional Context/Information

https://pastebin.com/D8WnkbNu
pastebin to the errors

### Make sure to confirm these as well before submitting the report:

- [X] The issue persists on the latest version of the mod.
- [X] All version numbers are properly recorded, no instances of ‘latest’
- [X] All details are reported clearly and concisely
- [X] This issue is not a duplicate of another issue
- [X] If the issue is a modpack-related issue, it only occurs *with* Redux installed, and not without
- [X] If the issue is a modpack-related issue, it has been reported to the modpack devs as well

**Comment by Ernesto-Velasco-Aguilar** [comment]
Having a similar error too. I don't have Aether Genesis however (unless that is base Aether?) 
Posting a log where it happened and had the same error message:
[2024-06-13-1.log](https://github.com/user-attachments/files/15828514/2024-06-13-1.log)

Only relevant mod on this context I have is "music manager" which I used to set the cooldowns between musics to 1 second, been trying with other mods or other cooldowns and still get no music in the aether however, it works correctly on other dimensions

EDIT: Turns out it doesn't work even without such mods, Redux simply stops all music in the aether, even in singlerplayer

**Comment by nocturni** [comment]
i'll add more to my initial post, i also used Music Manager to force music every 1 second to force these messages to appear, but the issue also happens without MM

**Comment by Ernesto-Velasco-Aguilar** [comment]
Did some testing, kept going back versions until I found one where it works (version 2.0.11), on any versions afterwards the issue persists. So perhaps 2.0.12's rewrite of the overlap system caused this bug

**Comment by Zepalesque** [comment]
Hmm, odd
I can't seem to reproduce the issue, strange
(at least in a dev environment, I'll check in a production environment in a modpack too though)

**Comment by Zepalesque** [comment]
yeah so i forgot to say this but i can't reproduce the issue on a full server either
